SPEC.md

```diff
@@ -533,3 +533,4 @@ The `dns` field contains a dictionary consisting of common DNS information.
 ## Well-known Error Codes
 - `1` - Incompatible CNI version
 - `2` - Unsupported field in network configuration. The error message must contain the key and value of the unsupported field.
+- `100` - Error during invocation of cni plugin. Plugin didn't specify the error code.
```

Comment:
Thanks for your reply and I have changed the commit log.
I think it's more proper for you or other members to add the above description to the SPEC. My understanding is limited to how it is used in the code and `100` may be used in other libs that implement the CNI specification :D